apps/user-profile/src/assets/config/defaultConfig.json

```diff
@@ -113,6 +113,22 @@
   ],
   "header_label_en": "Generic user profile",
   "header_label_cs": "Generický profil uživatele",
+  "log_out_enabled": true,
+  "admin_gui_label_en": "Admin GUI",
+  "admin_gui_label_cs": "Admin GUI",
```

Comment:
Not sure if something like `IDM Administration` or `Administration` wouldn't be more clear to users? You can try to think of something on your own. I just don't think the `Admin GUI` is very user-friendly.

Reply:
I changed labels to `IAM Administration`/`Správa IAM`, but I'm not still sure, if it is the best solution..